config/default.yaml

```diff
@@ -8,8 +8,10 @@ services:
   name: 'IDAM user dashboard'
   idam:
     url:
-      public: 'https://idam-web-public.aat.platform.hmcts.net'
-      api: 'https://idam-api.aat.platform.hmcts.net'
+        public: 'http://localhost:9000'
+        api: 'http://localhost:7060'
```

Comment:
This should be changed back to idam aat. In the PR environment, it doesnt know what is localshot:9000 and localhost:7060 so your test will fail to log in to Idam web public